lib/factory_girl.rb

```diff
@@ -43,7 +43,9 @@
 require 'factory_girl/decorator/disallows_duplicates_registry'
 require 'factory_girl/decorator/invocation_tracker'
 require 'factory_girl/decorator/new_constructor'
+require 'factory_girl/linter_base'
 require 'factory_girl/linter'
+require 'factory_girl/linter_with_traits'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.